We need to increase the number of dependency rescans as we have a large
number of self-referencing dependencies within Poky.  100 seem reasonable.

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

diff -ur rpm-5.1.10.orig/lib/depends.c rpm-5.1.10/lib/depends.c
--- rpm-5.1.10.orig/lib/depends.c	2010-08-20 17:12:43.000000000 -0500
+++ rpm-5.1.10/lib/depends.c	2010-09-02 13:37:21.966116309 -0500
@@ -2310,7 +2310,7 @@
     int numOrderList;
     int npeer = 128;	/* XXX more than deep enough for now. */
     int * peer = memset(alloca(npeer*sizeof(*peer)), 0, (npeer*sizeof(*peer)));
-    int nrescans = 10;
+    int nrescans = 100;
     int _printed = 0;
     char deptypechar;
     size_t tsbytes;
